### PR TITLE
Bump Formation to receive updates to side-nav

### PR DESCRIPTION
**Pull request by ncksllvn**
## Description
To receive the updates from https://github.com/department-of-veterans-affairs/design-system/pull/214.

### Todo
- [x] Remove the temporary code that loads our local sidenav SCSS file
- [x] Make sure https://github.com/department-of-veterans-affairs/design-system/pull/214 is merged
- [x] Update our version of the Design System via Yarn

## Testing done
Local testing

## Screenshots
Available in https://github.com/department-of-veterans-affairs/design-system/pull/214

## Acceptance criteria
- [x] Designer sign-off

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/platform/site-wide/sass/modules/_m-homepage.scss [hunk 1]
```diff
@@ -9,14 +9,6 @@
...
 }
 
-.homepage-hub, .va-sidebarnav, .vet-toolbar {
-  [href^="http"]:not(.usa-button-primary), [rel="external"] {
-    background-image: none;
-    padding-right: 0px;
-
-  }
-}
-
```
Comment (ncksllvn) [inline src/platform/site-wide/sass/modules/_m-homepage.scss]: This was an override to remove the external link icon, but that was already done in the design system here, https://github.com/department-of-veterans-affairs/design-system/pull/198.

**Review by rianfowler — APPROVED** [review]

**Review by emilywaggoner — CHANGES_REQUESTED** [review]
Thanks @ncksllvn! This is looking awesome. I have a couple of requests:

- I noticed that when hovering over the active level page item, the left border color changes to blue. Could it stay black instead?
- Also on the active level page item, would you mind changing the left padding from 25px to 12px?

Thank you!

**Comment by ncksllvn** [comment]
Thanks @emilywaggoner, we're making the updates now at https://github.com/department-of-veterans-affairs/design-system/pull/215

**Comment by ncksllvn** [comment]
@emilywaggoner We made those tweaks just now. Can you take another look? The Heroku review instance will refresh in a couple minutes.

**Review by emilywaggoner — APPROVED** [review]

**Comment by ncksllvn** [comment]
Thank you @emilywaggoner!

**Pull request merged**